qutebrowser/html/startpage.html

```diff
@@ -0,0 +1,179 @@
+    color: #bccbda;
+}
+
+.bookmarks {
+    display: flex;
+    flex-direction: column;
+}
+
+.bookmark-container {
+    border-color: #222d32;
+    background-color: #222d32;
+    border-radius: 8px;
+    border-width: 1px;
+    border-style: solid;
+    margin-bottom: 15px;
+    padding: 10px;
+}
+
+.bookmark-container-title {
+    margin: 0;
+    font-size: 25px;
+    font-family: monospace;
+    padding: 10px 10px 0 10px;
+    color: #84b4e6;
+}
+
+.link-container {
+    margin: 10px 15px;
+    font-size: 18px;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:link, a:visited {
+    color: #bdc4d4;
+}
+
+@media (prefers-color-scheme: light) {
+    body {
+        background-color: #ffffff;
+    }
+
+    .title {
+        color: #3cb0f3;
+    }
+
+    input {
+        background-color: #f4f7fffc;
+        color: #4380c3;
+        box-shadow: 0 2px 6px rgba(0, 0, 0, 0.15);
+    }
+
+    input::placeholder {
+        color: #7b87a5;
+    }
+
+    .bookmark-container {
+        margin-bottom: 20px;
+        background-color: #f4f7ff66;
+        border-color: #f4f7ff66;
+        box-shadow: 0 2px 6px rgba(0, 0, 0, 0.15);
+    }
+
+    .bookmark-container-title {
+        color: #4b91da;
+    }
+
+    a:link, a:visited {
+        color: #4b5a80;
+    }
+
+}
+{% endblock %}
+
+{% block content %}
+<div class="wrapper">
+  <div class="header">
+    <div class="title-container">
+      <img class="logo" src="{{ resource_url('img/qutebrowser_logo.svg') }}">
+      <span>
+        <h3 class="title">qutebrowser</h3>
+      </span>
+    </div>
+    <div class="search-box title-container" id="search">
+      <form>
+        <input type="text" size="50" name="q" placeholder="Search the web" id="search-field"/>
+        <input type="submit" style="display: none;" formaction="{{ search_url }}" formmethod="get" >
+      </form>
+    </div>
+  </div>
+  <div class="bookmarks">
+    {% if bookmarks %}
+    <div class="bookmark-container">
+      <h4 class="bookmark-container-title">Bookmarks</h4>
+      {% for bookmark in bookmarks %}
+      <div class="link-container">
+        <a class="bookmark-name" href="{{ bookmark[0] }}"> {{ bookmark[1] }} </a>
+      </div>
+      {% endfor %}
+    </div>
+    {% endif %}
+    {% if quickmarks %}
+    <div class="bookmark-container">
+      <h4 class="bookmark-container-title">Quickmarks</h4>
+      {% for quickmark in quickmarks %}
+      <div class="link-container">
+        <a class="bookmark-name" href="{{ quickmark[1] }}"> {{ quickmark[0] }} </a>
+      </div>
+      {% endfor %}
+    </div>
+    {% endif %}
+    <div class="bookmark-container">
+      <h4 class="bookmark-container-title">Getting Started</h4>
+      <div class="link-container">
+        <a class="link-name" href="https://qutebrowser.org">Official Website</a>
+      </div>
+      <div class="link-container">
+        <a class="link-name" href="https://raw.githubusercontent.com/qutebrowser/qutebrowser/master/doc/img/cheatsheet-big.png">Cheat Sheet</a>
```

Comment:
This could link to `qute://help/img/cheatsheet-big.png` instead - though perhaps it'd be better to link to the quickstart page (`qute://help/quickstart.html`) anyways?